content/rs/databases/active-active/aa-planning.md

```diff
@@ -0,0 +1,82 @@
+---
+Title: Considerations for planning Active-Active databases
+linktitle: Planning considerations
+description: Information about Active-Active database to take into consideration while planning a deployment, such as compatibility, limitations, and special configuration.
+weight: 22
+alwaysopen: false
+categories: ["RS"]
+aliases: [
+/rs/databases/active-active/aa-planning.md,
+/rs/databases/active-active/aa-planning/,
+]
+
+---
+
+In Redis Enterprise, Active-Active geo-distribution is based on [CRDT technology](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type) (conflict-free replicated data type). Compared to databases without geo-distribution, Active-Active databases have more complex replication and networking, as well as a different data type.
+
+Because of the complexities of Active-Active databases, there are  special considerations to keep in mind while planning your Active-Active database.
+
+See [Active-Active Redis]({{<relref "/rs/databases/active-active/">}}) for more information about geo-distributed replication. For more info on other high availability features, see [Durability and high availability]({{<relref "/rs/databases/configure/durability-ha.md">}}).
+
+## Participating clusters
+
+For Active-Active databases, you need to [set up your participating clusters]({{<relref "/rs/clusters/new-cluster-setup.md">}}). You need at least two participating clusters, but we recommend contacting Redis support for databases with more than ten. You can [add or remove participating clusters]({{<relref "/rs/databases/active-active/manage-aa#participating-clusters/">}}) after database creation.
+
+Changes made from the admin console to an Active-Active database configuration only apply to the cluster you are editing. For global configuration changes across all clusters, use the `crdb-cli` command-line utility.
+
+## Memory limits
+
+Database memory limits define the maximum size your database can reach across all database replicas and [shards]({{<relref "rs/concepts/terminology#redis-instance-shard">}}) on the cluster. Your memory limit will also determine the number of shards you'll need.
+
+Besides your dataset, the memory limit must also account for replication, Active-Active overhead, and module overhead. These features can significantly increase your database size, sometimes increasing it by four times or more.
```

Comment:
I think this fourfold increase could use an example or consider omitting the number.

Reply:
can we use a more "friendly" term for "overhead" ? maybe metadata/data/info or something in that spirit.
I would also change the second sentence to be less intimidating:
These features can increase your database size, sometimes increasing it by **two** times or more.